packages/@aws-cdk/aws-stepfunctions/lib/states/parallel.ts

```diff
@@ -59,6 +60,16 @@ export interface ParallelProps {
    * @default - None
    */
   readonly resultSelector?: { [key: string]: any };
+
+  /**
+   * Parameters pass a collection of key-value pairs as input. Values can be literals or JSONPath expressions that select from the step input.
+   *
+   * @see
+   * https://docs.aws.amazon.com/step-functions/latest/dg/input-output-inputpath-params.html#input-output-parameters
+   *
+   * @default - No parameters
+   */
+  readonly parameters?: { [key: string]: any };
```

Comment:
Is this mutually exclusive with `inputPath`? Should we not add a check that users aren't specifying both?

Reply:
Nevermind, it's not mutex.